command/polybftsecrets/params_test.go

```diff
@@ -59,3 +61,31 @@ func fileExists(filename string) bool {
 
 	return !info.IsDir()
 }
+
+func Test_getResult(t *testing.T) {
+	// Creates test directory
```

Comment:
Add parallel execution?

```suggestion
func Test_getResult(t *testing.T) {
        t.Parallel()
	// Creates test directory
```

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1248/commits/1f91d332597e272574c10bd34c06538e81735188